lib/static.js

```diff
@@ -33,21 +33,25 @@ function guineaPig (req, res) {
       domain: '.blargimarg.com',
       path: '/'
     });
-    res.send(getTemplate('guinea-pig.html')(params));
+    res.send((await getTemplate('guinea-pig.html'))(params));
   }, delay);
 }
 
 /*
  * Dynamic page mapped to /welcome
  */
-function welcome (req, res) {
+async function welcome (req, res) {
   let params = {message: 'Let\'s browse!'};
   log.debug(`Sending welcome response with params: ${JSON.stringify(params)}`);
-  res.send(getTemplate('welcome.html')(params));
+  res.send((await getTemplate('welcome.html'))(params));
+  return 'done!';
 }
 
-function getTemplate (templateName) {
-  return swig.compileFile(path.resolve(STATIC_DIR, 'test', templateName));
+async function getTemplate (templateName) {
+  // swig-/mustache-like {{}} templating
+  _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
```

Comment:
this is fine, or we could just switch the template to use lodash's `<%= content %>`.